pkg/shell-operator/debug_server.go

```diff
@@ -0,0 +1,97 @@
+package shell_operator
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/go-chi/chi"
+
+	"github.com/flant/shell-operator/pkg/app"
+	"github.com/flant/shell-operator/pkg/config"
+	"github.com/flant/shell-operator/pkg/debug"
+	"github.com/flant/shell-operator/pkg/task/dump"
+)
+
+func DefaultDebugServer() *debug.Server {
+	dbgSrv := debug.NewServer()
+	dbgSrv.WithPrefix("/debug")
+	dbgSrv.WithSocketPath(app.DebugUnixSocket)
```

Comment:
You have changed it [here](https://github.com/flant/shell-operator/pull/388/files#diff-9390d7fe36fa084015519d7717a8b8b3d1ae1c3c73f888705ee3903fef61793cR40)
But left `DebugUnixSocket` here. Is it ok?